### PR TITLE
BITMAKER-2338 estela-api: Create OwnerProject Role on user permissions

### PR DESCRIPTION
**Pull request by raymond1242**
# Description

estela-api: Create OwnerProject Role on user permissions

# Issue

https://tasks.bitmaker.dev/issues/2338?tab=time_entries

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] My code follows the style guidelines of this project.
- [x] I have made corresponding changes to the [documentation](https://github.com/bitmakerla/estela/tree/main/docs).
- [x] New and existing tests pass locally with my changes.
- [x] If this change is a core feature, I have added thorough tests.
- [x] If this change affects or depends on the behavior of other _estela_ repositories, I have created pull requests with the relevant changes in the affected repositories. Please, refer to our [official documentation](https://estela.bitmaker.la/docs/).
- [x] I understand that my pull request may be closed if it becomes obvious or I did not perform all of the steps above.

### CODE REVIEW

**Comment by raymond1242** [comment]
@JoseArmandoChavezQuijahuaman This PR is ready for review

File: estela-web/src/components/SpiderDetailPage/index.tsx [hunk 1]
```diff
@@ -707,9 +707,7 @@ export class SpiderDetailPage extends Component<RouteComponentProps<RouteParams>
                                             <Col className="float-right">
                                                 <Button
                                                     onClick={() => {
-                                                        history.push(
-                                                            `/projects/${this.projectId}/spiders/${this.spiderId}/jobs/create`,
-                                                        );
+                                                        history.push(`/projects/${this.projectId}/jobs`);
```
Comment (JoseArmandoChavezQuijahuaman) [inline estela-web/src/components/SpiderDetailPage/index.tsx:710]: Instead to redirect to this route, I think we can show the window to create a job

**Review by JoseArmandoChavezQuijahuaman — CHANGES_REQUESTED** [review]

File: estela-web/src/components/SpiderDetailPage/index.tsx (hunk 1 above)
Comment (raymond1242) [reply estela-web/src/components/SpiderDetailPage/index.tsx]: Solved

**Review by JoseArmandoChavezQuijahuaman — APPROVED** [review]
Good job, looks weel

**Pull request merged**